package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "spell": "yaspeller *.md",
     "clean": "rimraf api.md coverage/ test/demo/build/ test/demo/.cache",
-    "test": "jest --coverage && eslint . && check-dts && size-limit && yarn spell && yarn build",
+    "test": "jest --coverage && eslint . && check-dts && tsc --noEmit && size-limit && yarn spell && yarn build",
```

Comment:
Why do we need to run a compiler?

Reply:
The compiler will check that all types, imports, and dependencies are correct, or it will fall with an error.

Reply:
Like, check-dts

Reply:
Why these errors will not be found in `jest` call?

Reply:
Weeeell. Because ts-jest can affect how TS runs: for example, with a cache, it may not run all tests. Manual start of typechecking will be performed on clean settings and exactly on all files.

We can say that this is an action of paranoia.

Reply:
Let’s remove it. 3 different TS checker is too much :D.